src/qml/pages/onboarding/Onboarding00Welcome.qml

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+// The Onboarding00Welcome page.
+
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+
+Column {
```

Comment:
21c969272e718e63d3de00a9fe9e0f702eb4fd9f

We should extend `Pane`, something like:

```qml
Pane {
    contentItem: ColumnLayout {
        // ...
    }
}
```

Reply:
> We should extend `Pane`...

Why? We already have `ApplicationWindow` container.

Reply:
[Updated](https://github.com/bitcoin-core/gui-qml/pull/43#issuecomment-933995134).